### DISCUSSION TITLE
Content-Type for CRUD operations

### DISCUSSION THREAD

**Discussion by louiskhchan** [mockoon/mockoon#1255] [General]
Hi,

Our company uses Oracle server to create items by making POST requests with JSON objects. The requests have a `Content-Type: application/vnd.oracle.adf.resourceitem+json` header, and the server creates the item based on the JSON object and responds with the same content type.

We are using Mockoon to mock this operation. However, Mockoon’s CRUD route only accepts `application/json` content type to parse the input item as JSON. Otherwise, the input item is parsed as a string. This means we have to modify our app and change the content type, which is not ideal.

Does Mockoon have any plans to support other content types like `application/vnd.oracle.adf.resourceitem+json`? Or a setting to allow CRUD routes to accept other content types as JSON? Thank you.

**Comment by 255kb** [comment]
Hi, the condition for mockoon to parse entering JSON or served responses, is the presence of a compatible Content-Type header. It wouldn't be complicated to add a regex like `/application/.*\+json/` (didn't tried) to the list, to allow all the vendor specific Content-Types. 

Could you open an issue to track this? 
If you are willing to contribute, help is welcome 🙂

**Reply by louiskhchan** [reply]
I opened an issue here: https://github.com/mockoon/mockoon/issues/1269
Thank you.
